src/participant/participant.ts

```diff
@@ -449,24 +449,30 @@ export default class ParticipantController {
     ) as Promise<boolean>;
   }
 
-  async getDatabasesTree({
+  async renderDatabasesTree({
     command,
     context,
+    stream,
   }: {
     command: ParticipantCommand;
     context: vscode.ChatContext;
-  }): Promise<vscode.MarkdownString[]> {
+    stream: vscode.ChatResponseStream;
+  }): Promise<void> {
     const dataService = this._connectionController.getActiveDataService();
     if (!dataService) {
-      return [];
+      return;
     }
 
+    stream.push(
+      new vscode.ChatResponseProgressPart('Fetching database names...')
```

Comment:
So does the progress bar disappear as soon as we call `stream.markdown` or a similar output method?

Reply:
Yes